webviz_subsurface/containers/_reservoirsimulationtimeseries.py

```diff
@@ -0,0 +1,763 @@
+                                          for i in self.vctr_cols_no_hist],
+                                 value=self.vctr_cols_no_hist[0]),
+                    html.Div([
+                        dcc.Checklist(
+                            id=self.chlst,
+                            options=[{
+                                'label': 'Delta time series',
+                                'value': 'show_delta_time_series'
+                            }],
+                            labelStyle={'display': 'inline-block'},
+                            value=[],
+                        ),
+                        html.Div(id=self.show_ens_selectors, children=[
+                            dcc.Dropdown(
+                                id=self.dropdown_iorens_id,
+                                placeholder='Base case',
+                                options=[{'label': i, 'value': i}
+                                         for i in self.base_ensembles]
+                            ),
+                            dcc.Dropdown(
+                                id=self.dropdown_refens_id,
+                                placeholder='Select ensembles',
+                                options=[{'label': i, 'value': i}
+                                         for i in self.delta_ensembles],
+                                multi=True,
+                            ),
+                        ], style={'display': 'none'}),
+                    ]),
+                ], style={'width': '20%', 'float': 'left'}),
+
+                html.Div([
+                    dcc.Tabs(id=self.tab_id, value='summary_data', children=[
+                        dcc.Tab(label='Realizations', value='summary_data'),
+                        dcc.Tab(label='Statistics', value='summary_stats'),
+                    ]),
+                    html.Div(id='tabs-content'),
+                    html.Div(id=self.chart_id)
+                ], style={'width': '80%', 'float': 'right'}),
+
+            ]),
+        ])
+
+
+# =============================================================================
+# Callbacks
+# =============================================================================
+
+    def set_callbacks(self, app):
+
+        @app.callback(Output(self.show_ens_selectors, 'style'),
+                      [Input(self.chlst, 'value')])
+        def _func_show_ens_selectors(chlst: list):
+            """callback to update the styling of div that includes the ensemble
+            selectors. The styling switches to hiden when Delta time series is
+            not selected.
+
+            Input:
+                checklist values: list of strings = selected options
+            Output:
+                html.Div(...styling): dictionary describing styling
+            """
+
+            if 'show_delta_time_series' in chlst:
+                return {}
+            else:
+                return {'display': 'none'}
```

Comment:
This could be a one-liner.
`return {} if 'show_delta_time_series' in chlst else {'display': 'none'}`